metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -255,10 +244,6 @@ class DBTConfig(StatefulIngestionConfigBase):
         default=False,
         description="Use model identifier instead of model name if defined (if not, default to model name).",
     )
-    node_type_pattern: AllowDenyPattern = Field(
```

Comment:
Removing this requires updating the `updating.md` doc as this is a backwards incompatible change.